syntax = "proto3";

// 爬虫的的基本数据格式
message SpiderMessage{
enum SpiderStatus {
    // 刚创建还未请求的时候
    NONE= 0;
    // 需要重试
    RETRY = 1;
    // 完成
    FINISH = 2;
  }
enum RequestMethod {
    GET= 0;
    POST = 1;
  }
// 唯一id
string id = 1;
// 目标url
string target_url = 2;
// 解析配置
string parse_setting_id = 3;
// 创建时间戳
int64 create_time = 4;
SpiderStatus status = 5;
// 重试时间
int64 retry_time = 6;
// 解析时间
int64 finish_time = 7;
// 请求方法
RequestMethod request_method = 8;
// 步骤:根据解析配置中的步骤计算
int32 step = 9;
}


